services/nutanix-ai/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+displayName: Nutanix AI
+description: NAI core components.
+allowMultipleInstances: false
+category:
+  - Data Services
```

Comment:
Artificial Intelligence would be a better category name?

Reply:
I think it is a better name.